### DISCUSSION TITLE
SolidJS port

### DISCUSSION THREAD

**Discussion by shinebayar-g** [radix-ui/primitives#1919] [Ideas]
Radix UI is awesome. Is there any interest in maintaining SolidJS version?

**Comment by CanRau** [comment]
I'm quite new to solid and don't have a lot of time currently but this would be amazing and would also bring https://ui.shadcn.com/ to the solid community 🎉

**Reply by Koning3ton** [reply]
https://www.solid-ui.com/

**Comment by ZanzyTHEbar** [comment]
Any progress on this, i'd be happy to take part in the development here

**Comment by dinesh-hills** [comment]
Do include me as well

**Comment by szamanr** [comment]
found this, maybe people here can join forces: https://github.com/TrentsPC/solid-radix

there is also [Spectrum Web Components](https://opensource.adobe.com/spectrum-web-components/getting-started/) library, which uses web components and is framework agnostic.

**Comment by szamanr** [comment]
i also found Ark UI: https://ark-ui.com/docs/overview/introduction

works with react, solid and vue.

**Comment by naranyala** [comment]
https://github.com/stefan-karger/solid-ui

https://github.com/kobaltedev/kobalte

https://github.com/corvudev/corvu/
